js/templates/miniProfile.html

```diff
@@ -0,0 +1,16 @@
+<div class="flexVCent">
+  <div>
+    <div class="userIcon disc clrBr2 clrSh1" style="<%= ob.getAvatarBgImage(ob.avatarHashes || {}) %>"></div>
+  </div>
+  <div class="flexExpand">
+    <h1 class="h2 txUnb txUnl"><%= ob.name %></h1>
+    <div class="txt5b gutterHSm">
+      <span class="clrT">
+        <%= ob.parseEmojis('📍', '', { style: 'width: 10px' }) %> <%= ob.location || ob.polyT('userPage.noLocation') %>
```

Comment:
Similar to what I've done here... we're going to want to adjust other areas of the app that use a hard-coded emoji to pipe it through twemoji. Otherwise we risk that certain OSs / browsers will not render the emoji properly.